src/components/components/Slideshow.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div id="slideshow">
+    <ul>
+      <li v-for="(p, i) in pictures">
+        <img v-bind:src="p.url" v-bind:alt="p.alt" @click="setCurrent(i)" v-bind:class="'picture ' + (p.active ? 'active' : '')">
```

Comment:
`p.active` is equivalent to `i === currentPictureIndex`. No need for `active` prop